examples/runqslower/src/main.rs

```diff
@@ -8,6 +8,7 @@ use libbpf_rs::PerfBufferBuilder;
 use plain::Plain;
 use structopt::StructOpt;
 
+#[path = "bpf/.output/mod.rs"]
```

Comment:
does it have to be called generic "mod.rs"? What will happen if application is using two different skeletons? Ideally each skeleton should be called with its name, something like `runqslower.skel.rs` and `some_other.skel.rs`

Reply:
That is controlled in the build script: https://github.com/libbpf/libbpf-rs/blob/699e2a7e6b3e49e212902ce95314f850e2ffdf4e/examples/runqslower/build.rs#L16-L17

If there's multiple skeletons, the user is free to name each one whatever they want. That's the nice thing about the `#[path=..]` attribute

Reply:
cool, can we please use `runqslower.skel.rs` to set a good example?

Reply:
Done